synapse/handlers/room.py

```diff
@@ -203,37 +230,48 @@ def create(etype, content, **kwargs):
             },
         )
 
+        power_level_content = {
+            "users": {
+                creator.to_string(): 100,
+            },
+            "users_default": 0,
+            "events": {
+                EventTypes.Name: 100,
+                EventTypes.PowerLevels: 100,
+                EventTypes.RoomHistoryVisibility: 100,
+            },
+            "events_default": 0,
+            "state_default": 50,
+            "ban": 50,
+            "kick": 50,
+            "redact": 50,
+            "invite": 0,
+        }
+
+        if config["everyone_ops"]:
+            for invitee in invite_list:
+                power_level_content["users"][invitee] = 100
```

Comment:
I think it's a little unclear that `everyone_ops` really just means the room creator and the people in the *initial* invite list, no one else who subsequently joins/is invited. Surely if we really mean "everyone" then `users_default` should be `100`?